front/src/applications/stdcm/components/StdcmForm/StdcmConsist.tsx

```diff
@@ -102,6 +107,13 @@ const StdcmConsist = ({ isDebugMode, disabled = false }: StdcmConsistProps) => {
   const lengthFieldStatus = createFieldStatus('totalLength');
   const speedFieldStatus = createFieldStatus('maxSpeed');
 
+  const getMissingFieldMessage = (value?: number): string | undefined => {
+    if (value === undefined || value === null) {
+      return t('consist.errors.missingValue');
+    }
+    return undefined;
```

Comment:
Just a personal opinion but I find it better to return `null` in this case than `undefined`. It's that we will be "missing" something but we want to return something that isn't a string.
```suggestion
    return null;
```

Reply:
I agree, changed.